### PR TITLE
feat(caniuse): support output format parameter

### PR DESCRIPTION
**Pull request by sidvishnoi**

### CODE REVIEW

File: routes/caniuse/index.js [hunk 1]
```diff
@@ -41,5 +42,5 @@ async function route(req, res) {
 
   // cache for 24hours (86400 seconds)
   res.set("Cache-Control", "max-age=86400");
-  res.json(body);
+  res.send(body);
```
Comment (sidvishnoi) [inline routes/caniuse/index.js]: `res.send()` ensures JS objects are sent has `application/json` and strings go as `text/html`

Comment (marcoscaceres) [reply routes/caniuse/index.js]: oh nice... good that it knows how to detect the difference!

**Comment by marcoscaceres** [comment]
After this land, we should hit up Jake first for review, then maybe Simon Peters or other folks who know the image fetching/processing mechanics and give constructive feedback.

**Comment by marcoscaceres** [comment]
oops, commented in the wrong PR.

**Pull request merged**